app-script-payslip-sender/payslip-sender.gs

```diff
@@ -0,0 +1,28 @@
+function paySlipSender() {
+  var spreadsheet = SpreadsheetApp.getActive().getSheetByName("List");
+  var dataRange = spreadsheet.getRange("A2:D6").getValues();
```

Comment:
What if the data range is beyond 6th row? Need to handle this case also with a check for non-empty values.